sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts

```diff
@@ -540,6 +560,338 @@ describe("ApplicationStudentsController(e2e)-getCompletedApplicationDetails", ()
           },
           assessmentTriggerType: application.currentAssessment.triggerType,
           hasBlockFundingFeedbackError: true,
+          eCertFailedValidations: [],
+        });
+    },
+  );
+
+  it(
+    "Should get application details with ecert failed validations array having disability status not confirmed when " +
+      "the calculated PDPPD status is true and the student's disability status is permanent disability and the offering intensity is part time.",
+    async () => {
+      // Arrange
+      const student = await saveFakeStudent(db.dataSource);
+      const msfaaNumber = createFakeMSFAANumber(
+        {
+          student,
+        },
+        {
+          msfaaState: MSFAAStates.Signed,
+        },
+      );
+      await db.msfaaNumber.save(msfaaNumber);
+
+      // Mock user services to return the saved student.
+      await mockUserLoginInfo(appModule, student);
+
+      const application = await saveFakeApplicationDisbursements(
+        appDataSource,
+        {
+          student,
+          msfaaNumber,
+        },
+        {
+          applicationStatus: ApplicationStatus.Completed,
+          offeringIntensity: OfferingIntensity.partTime,
+          firstDisbursementInitialValues: {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Pending,
+          },
+        },
+      );
+      const [firstDisbursement] =
+        application.currentAssessment.disbursementSchedules;
+
+      application.currentAssessment.workflowData.calculatedData.pdppdStatus =
+        true;
+      await db.studentAssessment.save(application.currentAssessment);
+
+      application.student.disabilityStatus = DisabilityStatus.Requested;
+      await db.student.save(application.student);
```

Comment:
The student has the `disabilityStatus` already exposed in its `initialValue`. Can you please provide the value while creating the student?
```ts
 const student = await saveFakeStudent(db.dataSource, undefined, {
    initialValue: {
      disabilityStatus: DisabilityStatus.Requested,
    },
  });
```